src/utils/prim-parsing.fs

```diff
@@ -131,11 +132,7 @@ module internal Implementation =
     //-------------------------------------------------------------------------
     // Read the tables written by FSYACC.  
 
-    type AssocTable(elemTab:uint16[], offsetTab:uint16[]) =
-        let cacheSize = 7919 // the 1000'th prime
-        // Use a simpler hash table with faster lookup, but only one
-        // hash bucket per key.
-        let cache = Array.zeroCreate<int> (cacheSize * 2)
+    type AssocTable(elemTab:uint16[], offsetTab:uint16[], cache:int[], cacheSize:int) =
```

Comment:
Style: spaces after the `:`

Reply:
trivia: the compiler error messages showing method signatures has no space after `:`.